packages/app/src/utils.ts

```diff
@@ -9,3 +9,10 @@ export function getAttributeValue(
 ): unknown {
   return entity.attributes?.find((attr) => attr.name === attributeName)?.value;
 }
+
+export function hasAttribute(
+  entity: Entity,
+  attributeName: NxAttribute | ImageAttribute
+) {
+  return entity.attributes.some((attr) => attr.name === attributeName);
```

Comment:
Moved from `vis-packs/nexus/utils.ts` and refactored in preparation for the fetching of attribute values - i.e. it no longer calls `getAttributeValue`, which was unnecessary -- checking the existence of the attribute is sufficient.